package.json

```diff
@@ -152,11 +152,14 @@
 	],
 	"main": "./out/extension.js",
 	"dependencies": {
-		"vscode-languageclient": "~5.2.1"
+		"jsonc-parser": "^2.2.0",
+		"tar": "^5.0.5",
```

Comment:
Where is `tar` used?

Reply:
Just read the source code again and I was never aware the usage of tar even in the original code. I removed it. Thanks.